optimum/bettertransformer/transformation.py

```diff
@@ -123,7 +123,10 @@ class BetterTransformer(object):
     # Original PR from: https://github.com/huggingface/transformers/pull/19553 adapted and wrapped in this script.
     """
 
-    @check_if_pytorch_greater_112()
+    @check_if_pytorch_greater(
+        "1.13",
```

Comment:
```suggestion
        "1.13.0",
```

Reply:
yes it's cleaner at least for understanding!